documentation/wiki/Mac-Debugging.md

```diff
@@ -0,0 +1,38 @@
+#Debugging with MacOS
+Say unit tests are failing from Microsoft.Build.Engine.UnitTests_netcoreapp2.1_x64
+* Open terminal 
+  * Hit command-space, type terminal, hit enter. Alternatively, hit magnifying glass (spotlight) in upper-right corner and search for terminal.)
+* Build something
+  * Navigation in terminal is similar to command prompt (cd), although you type `ls` in place of `dir`.
+  * **Use ./build.sh instead of .\build.cmd.**
+* Type `find . -name Microsoft.Build.Engine.UnitTests_netcoreapp2.1_x64.log`
+  * This should print out a path (from your current working directory) to the relevant log file.
+* Type `tail <path from previous step>`
+  * This prints out the last part of that file.
+  * You can also just open it normally from a finder window.
+  * This file contains the standard output from the last run.
+  * You may notice that the file ends with `=== COMMAND LINE ===` followed by a single (long) command line statement.
+* Copy the command line statement from the previous step. Remove the portion after the redirection (`>` character). You may notice that part redirects output to the file you’re viewing.
+  * The last part (`2>&1`) redirects standard error (using `2>`) to the same place as where standard out is going (`&1`), in this case this log file.
+* Append `./build.sh &&` to the beginning of the truncated command line statement.
```

Comment:
I'd separate the "inner loop iterating on a test with `build && runtest` from the "running a single test in isolation" concepts--it's probably more valuable to doc the former but it's easier to understand the latter.

Reply:
Do you mean that you'd separate the building step from the running test step? If so, that might confuse people who see a bug, fix the bug, run the test again, and get the same bug because they didn't rebuild.